buildenv

```diff
@@ -70,21 +70,26 @@ zopen_init()
 zopen_check()
 {
   WORK_DIR="${ZOPEN_ROOT}/work"
+  INCLUDE_DIR="${ZOPEN_ROOT}/tests/include"
   mkdir -p "${WORK_DIR}"
 
   #
   # Set up environment
   #
   . ${ZOPEN_ROOT}/meta/.env
 
+  # Display banner
+  ${INCLUDE_DIR}/banner.sh "Start Tests"
```

Comment:
Might be better to rename this to `common.sh`, and then you can source it:
```
. ${INCLUDE_DIR}/common.sh
```

and then call `zopen_print_banner "Start Tests"`

Reply:
This way we can add other common test library functions into common.sh